sdk/src/Core/Internal/Entities/TraceId.cs

```diff
@@ -115,15 +114,14 @@ public static bool IsIdValid(string traceId)
             }
 
             // Is the epoch a valid 32bit hex number?
-            int epoch;
-            if (!int.TryParse(idEpoch, NumberStyles.HexNumber, null, out epoch))
+            if (!int.TryParse(idEpoch, NumberStyles.HexNumber, null, out _))
             {
                 return false;
             }
 
             // Is the random number a valid hex number?
-            BigInteger rand = BigInteger.Zero;
-            if (!BigInteger.TryParse(idRand, NumberStyles.HexNumber, null, out rand))
+            _ = BigInteger.Zero;
```

Comment:
Always a great day to learn something new.  I'm glad you pointed this out however, because there is no point to assign `_ = BigInteger.Zero;`  The focus here was that the output was to be discarded, thus that line not necessary, as the out declaration on BigInteger.TryParse` can simply be discarded with `out _`.  That being said, I just pushed up some changes removing the unnecessary declarations.